examples/http-server/main_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func setupTestServer(t *testing.T) *httptest.Server {
+	t.Helper()
+	if err := setupCoraza(); err != nil {
+		panic(err)
+	}
+	return httptest.NewServer(corazaRequestHandler(http.HandlerFunc(hello)))
+}
+
+func doGetRequest(t *testing.T, getPath string) int {
+	t.Helper()
+	resp, err := http.Get(getPath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	defer resp.Body.Close()
+	return resp.StatusCode
+}
+
+func TestHttpServerTrueNegative(t *testing.T) {
```

Comment:
Don't think we need the word `True`.

Also I guess this is a good case for a Go table test, both to learn Go idioms and to save on starting a test server each time.

There aren't that many example yet though there probably should be more, but for example here

https://github.com/corazawaf/coraza/blob/fea0b3cd2662591bbdd3ee20c125e30f299e9b53/actions/severity_test.go#L14

Something like


```
tests := []struct{
  name string
  path string
  expStatus int
}{
{
  name: "positive",
  path: "/hello"
  expStatus: 200
}
}

testServer := setupTestServer(t)

for _, tc := range tests {
  tt := tc
  t.Run(tt.name, func() {
    code := doGetRequest(
    if code != tt.expStatus {
     
```